build.gradle

```
@@ -43,81 +43,30 @@ apply from: file( 'gradle/module.gradle' )
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 // Release Task
 
-task release {
+tasks.register('release') {
 	description = "The task performed when we are performing a release build.  Relies on " +
 			"the fact that subprojects will appropriately define a release task " +
 			"themselves if they have any release-related activities to perform"
 
 	doFirst {
 		def javaVersionsInUse = jdkVersions.allVersions
-		if ( javaVersionsInUse != [JavaLanguageVersion.of( 11 )].toSet() ) {
-			throw new IllegalStateException( "Please use JDK 11 to perform the release. Currently using: ${javaVersionsInUse}" )
+		if (javaVersionsInUse != [JavaLanguageVersion.of(11)].toSet()) {
+			throw new IllegalStateException("Please use JDK 11 to perform the release. Currently using: ${javaVersionsInUse}")
```

Should probably be 17 since we're on main, no?

```suggestion
		if (javaVersionsInUse != [JavaLanguageVersion.of(17)].toSet()) {
			throw new IllegalStateException("Please use JDK 17 to perform the release. Currently using: ${javaVersionsInUse}")
```

---

Probably.  But I did not change this